src/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.cs

```diff
@@ -447,7 +447,7 @@ public override string[] GetManifestResourceNames()
             return InternalAssembly.GetManifestResourceNames();
         }
 
-        public override FileStream GetFile(string name)
+        public override FileStream? GetFile(string name)
```

Comment:
>? [](start = 34, length = 1)

Are all of these nullable annotations on the returns of these methods accurate? e.g. AssemblyBuilder.GetFile can return null?  If so, ok.  If not, it'd be good to make them non-nullable as appropriate, and if the compiler complains, suppress the warning with a pragma and link to the relevant roslyn bug.

Reply:
only implelentation for this method is in RuntimeAssembly, others are throwing including one called here. Will make non nullable